src/Workspaces/CoreTest/Shared/Extensions/TelemetryExtensions/TelemetryExtensionTests.cs

```diff
@@ -5,10 +5,10 @@
 #nullable disable
 
 using System;
-using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
+using Microsoft.CodeAnalysis.Shared.Extensions;
 using Xunit;
 
-namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Extensions
+namespace Microsoft.CodeAnalysis.UnitTests.Extensions
```

Comment:
This namespace change means the expected TelemetryId has changed to `00000000-0000-0000-54ad-749900000000`. Also, it seems we are asserting the expected and actual values in the wrong order.